app/hedge.py

```diff
@@ -125,8 +125,9 @@ def main():
                 else:  # Runs video
                     print('Running Video')
                     try:
+                        path = Path(__file__).resolve().parents / 'video.py'
                         subprocess.check_output(
-                            ['python3', '/home/pi/v0.8.3/video.py'], timeout=120)
+                            ['python3', '/home/pi/HogPi/app/video.py'], timeout=120)
```

Comment:
you should be using the value of `path` here, i.e. `['python3', path]`

Reply:
will do